v0/destinations/freshsales/transform.js

```diff
@@ -48,6 +66,54 @@ const identifyResponseBuilder = (message, { Config }) => {
   return response;
 };
 
+/*
+ * This functions is used for tracking contacts activities.
+ * @param {*} message
+ * @param {*} Config
+ * @returns
+ */
+const trackResponseBuilder = async (message, { Config }) => {
+  const { event } = message;
+  if (!event) {
+    throw new CustomError("Event name is required for track call.", 400);
+  }
+  let payload;
+  const email = getFieldValueFromMessage(message, "email");
+  const response = defaultRequestConfig();
+  switch (
+    event
+      .toLowerCase()
+      .trim()
+      .replace(/\s+/g, "_")
+  ) {
+    case "sales_activity":
```

Comment:
Can we add mapping in UI 

Rudder Event  --->  Freshsales sales_activity or lifecycle_stage (drop down)

Reply:
I meant a UI mapping similar to our ads destination check SnapChat conversions for example

Reply:
Done